### PR TITLE
Format code using gofmt and add gofmt check in build pipeline , remove frontend and backend checks from push pieline

### PR DESCRIPTION
**Pull request by specter25**
## Proposed changes

-Format code using gofmt
-add gofmt check in build pipeline
- remove frontend and backend checks from push pieline
## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [x] New feature (non-breaking change which adds functionality)
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [x] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the commit for DCO to be passed.
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

## Dependency
- Please add the links to the dependent PR need to be merged before this (if any).

### CODE REVIEW

**Comment by specter25** [comment]
@gdsoumya sir i have configured the gofmt check here as you asked in this [comment ](https://github.com/litmuschaos/litmus/pull/2661#issuecomment-815668210)

**Comment by specter25** [comment]
@rajdas98 sir i have removed the the frontend and backend checks from the push pipeline as mentioned in this issue #2649 . Sir by the time we test golang-ci lint , i have added the go-fmt check as requested by @gdsoumya sir .

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -44,6 +44,10 @@ jobs:
         uses: Jerome1337/goimports-action@v1.0.3
         with:
           goimports-path: ./litmus-portal
+      - uses: Jerome1337/gofmt-action@v1.0.4
+        with:
+          gofmt-path: './litmus-portal'
+          gofmt-flags: '-l -d'
```
Comment (rajdas98) [inline .github/workflows/build.yml:50]: gofmt check is already present

Comment (rajdas98) [reply .github/workflows/build.yml:50]: In the Makefile

Comment (specter25) [reply .github/workflows/build.yml:50]: Sir but there was some unformatted code , which got formatted when i ran the gofmt command from the terminal

Comment (specter25) [reply .github/workflows/build.yml:50]: I am not sure is , it because of the go fmt version ?

Comment (specter25) [reply .github/workflows/build.yml:50]: Should i remove it then ?

**Comment by rajdas98** [comment]
Gofmt is already there in the Makefile. Closing this Pr

**Pull request closed without merging**